tests/tls/tls-issuer-zero/test.yaml

```diff
@@ -0,0 +1,12 @@
+requires:
+  min-version: 9
+
+args:
+- -k none
+
+checks:
+- filter:
+    count: 1
+    match:
+      anomaly.app_proto: tls
+      anomaly.event: CERTIFICATE_INVALID_ISSUER
```

Comment:
So, basically, engine handles it in a user friendly way but this is not ideal and can be made better. Next rev with that and Suricata PR

Reply:
Done what I wanted in the following rev